docs/product/stats/index.mdx

```diff
@@ -23,9 +14,82 @@ With the dropdowns at the top of the page, you can set whether the page displays
   event type.
 </Note>
 
+
+There are three tabs available on the page: [Usage](#usage-stats), [Issues](#issue-stats), and [Health](#health-stats).
+
+The "Usage" tab shows your organization’s event and attachments usage, while also providing a high-level overview of how different projects are using Sentry. The "Issues" tab shows a breakdown of the issues assigned to your team and how they triaged them, while the "Health" tab provides an overview of your team’s project health.
+
+
+## Usage Stats
+
+The "Usage" tab provides an overview of the events (errors and transactions) and attachments that Sentry has received across your entire organization. It includes a detailed breakdown of each project's consumption categorized as _Accepted_, _Filtered_, _Rate Limited_, _Invalid_ or _Client Discard_.
+
+### Accepted
+
+Events and attachments that were successfully processed and stored.
+
+### Filtered
+
+Filtered events and attachments intentionally excluded based on definied settings. The following reasons are currently defined:
+
+- **Browser Extensions**: Filtered by browser extension.
+- **Error Message**: Filtered based on specific error messages.
+- **Filtered Transaction**: Filtered because it was a call to a filtered transaction.
+- **Invalid CSP**: Filtered due to an invalid Content Security Policy (CSP) policy.
+- **Ip Address**: Filtered by IP address.
+- **Legacy Browsers**: Filtered by certain legacy versions of browsers that are known to cause problems.
+- **Localhost**: Filtered due to localhost restriction.
+- **Release Version**: Filtered by release name (version).
+- **Web Crawlers**: Identified as a known web crawler.
+- **Other**: Filtered due to other generic filtering rules not specified above.
```

Comment:
```suggestion
- **Browser Extensions**: Filtered by browser extension.
- **Error Message**: Filtered based on specific error messages.
- **Filtered Transaction**: Filtered because it was a call to a filtered transaction.
- **Invalid CSP**: Filtered due to an invalid Content Security Policy (CSP) policy.
- **Ip Address**: Filtered by IP address.
- **Legacy Browsers**: Filtered by certain legacy versions of browsers that are known to cause problems.
- **Localhost**: Filtered due to localhost restriction.
- **Release Version**: Filtered by release name (version).
- **Web Crawlers**: Identified as a known web crawler.
- **Other**: Filtered due to other generic filtering rules not specified above.
```